pkgs/games/lunar-client/default.nix

```diff
@@ -2,12 +2,12 @@
 
 let
   name = "lunar-client";
-  version = "2.15.1";
+  version = "2.16.1";
 
   desktopItem = makeDesktopItem {
     name = "lunar-client";
     exec = "lunar-client";
-    icon = "lunarclient";
+    icon = "launcher";
```

Comment:
Isn't that name super generic and could easily clash?

Reply:
Should be better now :)